### ISSUE TITLE
The latest 1.x releases are lying about their PHP version support

### ISSUE THREAD

**Issue by stof** [SymfonyTest/symfony-bundle-test#60]
The composer.json says that PHP 5.5+ is supported, but the code uses return types.

**Comment by stof** [comment]
Looks like only 1.8.1 is affected.

**Comment by chapterjason** [comment]
@Nyholm 

Since 3.4 isn't maintained, we should adjust the requirements to ^4.4.
Bundles should update their dependencies.

```diff
    "require": {
-        "php": "^5.5 || ^7.0 || ^8.0",
+        "php": ">=7.1.3 || ^8.0",
-        "symfony/dependency-injection": "^3.4 || ^4.3 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^3.4 || ^4.3 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^3.4 || ^4.3 || ^5.0 || ^6.0",
-        "symfony/yaml": "^3.4 || ^4.3 || ^5.0 || ^6.0"
+        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
+       "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0",
+        "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0",
+        "symfony/yaml": "^4.4 || ^5.0 || ^6.0"
    },
```

**Comment by stof** [comment]
Even then, you should either delete the 1.8.1 release (which is a pain as it is very easy to re-push the tag by mistake if it has already propagated to forks or local clones) or tag a 1.8.2 release reverting the issue (and then do a 1.9.0 or 1.8.3 re-applying the change but with a correct composer.json). this way, the latest release marked compatible with older PHP versions (and so the one used by composer during solving) would actually work.

**Comment by Nyholm** [comment]
I think we should do 1.8.2.

**Comment by Nyholm** [comment]
Hm,.. or it looks like I tagged the wrong branch. Hm..

**Comment by Nyholm** [comment]
This is now fixed in 1.8.2 and dev-1.x
